.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Test
+
+on:
+  push:
+    branches: [ "master" ]
```

Comment:
technically our repo uses main... not sure if we even want/need these workflows in our nexrender repo?